data-serving/data-service/src/index.ts

```diff
@@ -35,7 +35,7 @@ const app = express();
 dotenv.config();
 const env = validateEnv();
 
-const deployment_envs = ['prod', 'dev'];
+const deployment_envs = ['prod', 'dev', 'qa'];
```

Comment:
proper nitpicking, but I'd put qa in between dev and prod just as a reminder of the deployment order.

Reply:
> Looks good @jim-sheldon! I think qa should generally be prod like, with deployments from the tip of the latest stable branch.
> 
> I'd rename prod to prod-dev or sth similar, as right now prod/ contains both prod and dev deployments

Good call, I will rename the folder.
There are a few branching and deployment strategies we could adopt; we should discuss this at greater length on a call or in a meeting.